src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/7R_pz2Fz4qE" frameborder="0" allowfullscreen></iframe>
+
+#### AsTeRICS
+
+The <a href="https://github.com/asterics" target="_blank" class="link-external">Assistive Technology Rapid Integration &amp; Construction Set</a> is an open source construction set for assistive technologies.
+
+AsTeRICS can be used to rapidly prototype new forms of assistive technology using a graphical construction environment that runs on Windows or as a <a href="https://github.com/asterics/WebACS" target="_blank" class="link-external">web-based application.</a>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JCTK75cOqXk" frameborder="0" allowfullscreen></iframe>
+
+#### Inclusive Design Guide
+
+The <a href="https://guide.inclusivedesign.ca/" target="_blank" class="link-external">Inclusive Design Guide</a> is an evolving collection of insights, practices, tools and activities that can be applied a wide range of design challenges in the digital, physical and service realm.
+
+Some of the parts of the Inclusive Design Guide relevant to hacking and making events or activities include:
+
+* activities like <a href="https://guide.inclusivedesign.ca/activities/InclusiveDesignMapping.html" target="_blank" class="link-external">inclusive design mapping</a> and the <a href="https://guide.inclusivedesign.ca/activities/MatchingGame.html" target="_blank" class="link-external">matching game</a> help in conceiving new inclusive products, or improving existing ones.
+* the <a href="https://guide.inclusivedesign.ca/tools/UserStatesAndContexts.html" target="_blank" class="link-external">user states and contexts</a> use-modelling tool can be used to think about a proposed design through the lens of various different user contexts.
+*  using <a href="https://guide.inclusivedesign.ca/tools/MultimodalTesting.html" target="_blank" class="link-external">multimodal testing</a> to evaluate ideas or prototypes.
+
+Like this handbook, the Guide is an evolving resource, and new sections are added as we learn more about designing with inclusion in mind.
+
+#### The Developer Space
+
+The three examples above are only a sample of what is available; more can be found by exploring the <a href="http://dspace.teco.edu/" target="_blank" class="link-external">Developer Space</a>, especially the lists of <a href="http://dspace.teco.edu/en/categories/parts-tools" target="_blank" class="link-external">Parts &amp; Tools</a>.
+
+### Other Tools
+
+This is a far from comprehensive discussion of other tools that may be useful in inclusive learning settings for hacking and making.
+
+#### Specialized Programming Environments
+
+There is a wide range of software built with the intention of supporting more specific programming tasks than "general purpose" programming languages, or providing alternatives to the more familiar text-centric approaches to programming languages.
```

Comment:
Is there something to be said about how some "specific purpose" programming environments are designed to empower people who would otherwise not identify as programmers or do programming? (e.g. Scratch -> children, Processing -> artists, Max/MSP -> musicians, etc.)